mmv1/products/migrationcenter/PreferenceSet.yaml

```diff
@@ -0,0 +1,197 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+base_url: projects/{{project}}/locations/{{location}}/preferenceSets
+create_url: projects/{{project}}/locations/{{location}}/preferenceSets?preferenceSetId={{preference_set_id}}
+self_link: projects/{{project}}/locations/{{location}}/preferenceSets/{{preference_set_id}}
+id_format: projects/{{project}}/locations/{{location}}/preferenceSets/{{preference_set_id}}
+import_format:
+  - projects/{{project}}/locations/{{location}}/preferenceSets/{{preference_set_id}}
+name: PreferenceSet
+description: Manages the PreferenceSet resource.
+update_verb: :PATCH
+update_mask: true
+autogen_async: true
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'preference_set_basic'
+    primary_resource_id: 'default'
+    vars:
+      set_name: 'preference-set-test'
+properties:
+  - !ruby/object:Api::Type::String
+    name: name
```

Comment:
I don't want to rewrite these because it will cause a conflict with the API field called `name`. Usually that's not a big deal, but in the interest of operating these resources as a pass-through and not having to make overrides I think it's better to keep the API-defined name of the identifier

Reply:
Makes sense